sct_ssh.py

```diff
@@ -433,3 +443,99 @@ def gcp_allow_public(user, test_id):
                      project=info['project_id'],
                      zone=i.zone.split('/')[-1])
         click.echo(click.style(f"set netwrok tag 'sct-allow-public' to {get_name(i)}", fg='green'))
+
+
+@click.command("update-scylla-packages", help="Update ScyllaDB cluster packages ...")
+@click.option("--test-id", type=str, help='Find cluster by test-id', required=True)
+@click.option(
+    "-p", "--packages-location", type=str,
+    help='Location where new packages are placed. Can be local directory, s3:// or gs:// urls')
```

Comment:
it would be nice, if we could use a local path, and we'll take care of the upload to s3 ?

Reply:
never mind, I didn't fully read the description here, and I see it can do both, local folder, and one from cloud storage